### PR TITLE
Syndie MMIs are now compatible with AI cores

### PR DESCRIPTION
**Pull request by FluffySurvivor**
https://forums.yogstation.net/index.php?threads/make-syndicate-mmi-capable-of-fitting-into-ai-cores.8475/

Will hopefully make the surgery bag purchase not useless.

:cl: FluffySurvivor
tweak: Syndicate MMIs now fit into AI core frames.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
If you wanna make the bag not useless, make the bottle oldchloral or neurotoxin or pancuronium instead of morphine.

File: code/modules/mob/living/carbon/brain/MMI.dm [hunk 1]
```diff
@@ -165,5 +165,5 @@
 
 /obj/item/device/mmi/syndie
 	name = "Syndicate Man-Machine Interface"
-	desc = "Syndicate's own brand of MMI. It enforces laws designed to help Syndicate agents achieve their goals upon cyborgs created with it, but doesn't fit in Nanotrasen AI cores."
+	desc = "Syndicate's own brand of MMI. It enforces laws designed to help Syndicate agents achieve their goals upon cyborgs created with it, can fit into Nanotrasen AI cores as well."
```
Comment (Cruix) [inline code/modules/mob/living/carbon/brain/MMI.dm]: And can fit into

**Comment by X-TheDark** [comment]
What about Syndicate MMI's allowing the borg to select Syndicate Cyborg module. Would that make the bag not useless?

**Comment by FluffySurvivor** [comment]
The Syndieborg is worth 80 TC. Spending 4 tc to get a 80 TC death machine. 

In practice, it's an extremely powerful murderbone asset. Indeed, the whole process of debraining someone is risky, but it's easily offset when you get the syndieborg and stomp everyone.

**Comment by ghost** [comment]
Here's an idea.

Replace the syndicate surgery bag with a Syndicate Cyborg Frame and a standard MMI, in a box.

The Frame looks exactly like a normal cyborg frame, but on examination has the word 'SYNDICATE' stenciled into the top of it's head.

Upon inserting the brain, the cyborg gains standard emagged borg laws and emagged borg modules.

Basically, it means you don't have to break into robotics to make use of the item-it comes with the MMI and the cyborg frame, now your only difficulty is killing and debraining the guy.

**Comment by X-TheDark** [comment]
Should it be exoskeleton+parts (so you can assemble it when you want) or should it be the full frame (that cannot be put into backpack). Plus, the words SYNDICATE also appear on examination even after the borg is assembled and MMI put into it, correct?

EDIT: Either way, the rework is something that should be discussed separately and not in this PR, I think. Nothing against this fitting into AI.

**Comment by ghost** [comment]
no, not on examine after assemble

**Comment by ShadowDeath6** [comment]
Does the syndie mmi actually do things or is it just a renamed MMI?

**Comment by FluffySurvivor** [comment]
It inserts syndie laws.

**Comment by DavidEdwards** [comment]
Any additional comments?

**Comment by FluffySurvivor** [comment]
I'd say it's good to go

**Pull request merged**